packages/clay-dropdown/src/ClayDropdownItem.soy

```diff
@@ -278,7 +286,10 @@
 			{/if}
 		"
 
-		data-onclick="{$_handleItemClick}"
+		{if not $disabled}
+			data-onchange="{$_handleItemClick}"
+		{/if}
+
 		data-onkeydown="{$_handleItemKeyDown}"
```

Comment:
Why not `onkeydown`?

Reply:
For some reason we do not have `delegateTarget` when we only use `onkeydown`.